doc/api/util.md

```diff
@@ -3305,6 +3305,29 @@ util.isUndefined(null);
 // Returns: false
 ```
 
+### `util.loadEnvFile(path)`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1.1 - Active Development
+
+* `path` {string | undefined}
+
+Loads the `.env` file into `process.env`. By default the path is resolved to
+`path.resolve(process.cwd(), '.env')`. If file is not found, no error is thrown.
+
```

Comment:
I think the docs need some discussion of `NODE_OPTIONS`. By this point Node is already running, so many of the settings that can be defined in `NODE_OPTIONS` and work via `--env-file` won’t work via this API. So then what? None of them work? Whatever works if you do `process.env.NODE_OPTIONS = '...'` works here too? Something in between?

Reply:
NODE_OPTIONS will not work in here. Yes, we should document this properly. Nice catch.